File: TumCampusApp/src/de/tum/in/tumcampusapp/models/Geo.java
package de.tum.in.tumcampusapp.models;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "geo")
public class Geo {
	@Element(required = false)
	private String latitude="0";
	
	public String getLatitude() {
		return latitude;
	}

	@Element(required = false)
	private String longitude="0";

	public String getLongitude() {
		return longitude;
	}
	
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/other/locations/model/Geo.kt
package de.tum.`in`.tumcampusapp.component.other.locations.model

import com.tickaroo.tikxml.annotation.PropertyElement
import com.tickaroo.tikxml.annotation.Xml

@Xml(name = "geo")
data class Geo(
    @PropertyElement var latitude: String = "0",
    @PropertyElement var longitude: String = "0"
) {

    constructor(latitude: Double, longitude: Double) : this(
            latitude = latitude.toString(),
            longitude = longitude.toString()
    )

    override fun toString(): String = "latitude=$latitude longitude=$longitude"
}